todo_app/app.py

```diff
@@ -8,4 +9,11 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template('index.html', items=get_items())
+
+
+@app.route('/items', methods=["POST"])
+def add_items():
```

Comment:
I think this route is only creating 1 item, so I would prefer to be exact and not use a plural for the naming.